wdl/MainVcfQc.wdl

```diff
@@ -85,7 +88,20 @@ workflow MainVcfQc {
     }
   }
 
-  Array[File] vcfs_for_qc = select_first([SubsetVcfBySamplesList.vcf_subset, vcfs])
+  if (defined(sample_renaming_json)) {
```

Comment:
Do you think there's a way to do the renaming for just the external per-sample benchmarking, but not for the rest of the per-sample QC collection? That might be too complicated, because there's a lot more files, but if it was able to just map between the current cohort's sample IDs and the external cohort's sample IDs within the per-sample comparison script that seems like it would be ideal

Reply:
Honestly I think the thing to do is to rename the samples in the external comparison set: I went with this approach because it was suggested to me and I was too busy to sit and think through the implications. The problem with only renaming during the external per-sample benchmarking, is that it takes its data from the bed files produced by `CollectQcVcfWide`. It felt like I might have to intervene in names in multiple places, and maybe mess up somewhere.

If this is a significant problem, it could be a good reason to go back to the drawing board and instead rename the files in the external tarball.

Reply:
Do you think it could work to pass a sample renaming map to this script and use the ID that matches the external benchmarking dataset here when creating the per-sample BED files in the tarball? https://github.com/broadinstitute/gatk-sv/blob/cd5ce7e0a890c4c5234341e9fc976b4d56d3ca60/src/sv-pipeline/scripts/vcf_qc/collectQC.perSample_benchmarking.sh#L158